mailpoet/assets/js/src/automation/automation.tsx

```diff
@@ -79,24 +102,27 @@ function DeleteSchemaButton(): JSX.Element {
 
 function App(): JSX.Element {
   return (
-    <div>
-      <Workflows />
-      <div style={{ marginTop: 30, display: 'grid', gridGap: 8 }}>
-        <CreateTestingWorkflowButton />
-        <CreateWorkflowFromTemplateButton slug="simple-welcome-email">
-          Create testing workflow from template (welcome email)
-        </CreateWorkflowFromTemplateButton>
-        <CreateWorkflowFromTemplateButton slug="welcome-email-sequence">
-          Create testing workflow from template (welcome sequence, only premium)
-        </CreateWorkflowFromTemplateButton>
-        <CreateWorkflowFromTemplateButton slug="advanced-welcome-email-sequence">
-          Create testing workflow from template (advanced welcome sequence, only
-          premium)
-        </CreateWorkflowFromTemplateButton>
-        <RecreateSchemaButton />
-        <DeleteSchemaButton />
+    <BrowserRouter>
```

Comment:
I don't know if we want/need to wrap the entire App with the router

Reply:
My guess is the router only needs to sit above the highest place in the hierarchy where we're routing. That said, it's probably fine to have it somewhere very close to the root of the app or directly at the root, so I think this is good.